marlowe-playground-client/src/Halogen/Classes.purs

```diff
@@ -169,6 +169,12 @@ plusBtn = ClassName "plus-btn"
 minusBtn :: ClassName
 minusBtn = ClassName "minus-btn"
 
+upBtn :: ClassName
```

Comment:
we are trying to get away from this module and just use tailwind classes as string.

Reply:
I'm leaving this for now as it is - ok?